src/components/About.js

```diff
@@ -1,8 +1,34 @@
-import React from "react";
+import React from 'react';
 
 const About = () => {
   return (
-    <h1> Hello This is About page</h1>
+    <main className="mx-5 my-3">
+      <h2 className="text-uppercase">
+        <strong>About CYF Quiz Project</strong>
```

Comment:
`strong` is used for emphasising parts of a sentence, so it doesn't make sense to use it here — let's remove it. If we wanted to bold the heading then we would use CSS instead.

Reply:
Done